recipes/libpcsclite/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "libpcsclite" %}
+{% set version = "1.9.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/LudovicRousseau/PCSC/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 856a1daed9a5ae1ac265c75ce9f81743e66a972e5be5a1d3a2196ed23f75ea26
+
+build:
+  skip: True  # [win]
+  script:
+    - ./bootstrap
+    - ./configure  # [linux]
+    - ./configure --disable-libsystemd  # [osx]
+    - make
+    # Test is here
+    - make distcheck  # [linux]
```

Comment:
I don't think you are actually installing anything to $PREFIX. You're just building and testing. Probably you need to call `make install`.

Reply:
@carterbox I'll try calling make install instead of make? And see what happens

Reply:
@carterbox I might need to use certain options to make install into $PREFIX, right now it's trying to install into /usr/bin?

Reply:
Yes, you need to install the build artifacts to $PREFIX.

Reply:
Right but... how do I do that?

Reply:
I think when you call `./configure` you need to pass `--prefix=$PREFIX`, and then `make install` will work.

Reply:
It worked! Thanks